lib/move.ex

```diff
@@ -0,0 +1,25 @@
+defmodule Move do
+  def translate(move) do
+    find_letters_and_numbers(move)
+    |> build_coordinate_pair
+  end
+
+  defp find_letters_and_numbers(string) do
+    Regex.scan(~r/\d+|[a-zA-Z]/, string, capture: :all)
+  end
```

Comment:
I wasn't 100% sure what this function does so I tried finding a unit test that might be able to shed some light but I surprised there weren't any test cases around this logic.

Reply:
Thank you, @klkelley! I'll go back and add unit tests around this in the next PR I create :)